content/broker/broker-transition/how-to-get-started/_index.en.md

```diff
@@ -0,0 +1,32 @@
+---
+title: How To Get Started
+linktitle: How To Get Started
+description: How to get started using the Broker Transition solution from Altinn 2 to Altinn 3   
+tags: [architecture, solution]
+toc: false
+weight: 1
+---
+
+## ServiceOwners
+In order to use the Broker Transition solution in Altinn 2 to create, upload and retrieve file metadata, a service owner must complete the following steps.
+1. Have an existing Altinn 2 Broker Service.
+2. Have or create a corresponding Altinn 3 Broker Resource.
+3. Request a transition setup from the Altinn 2 Service to the Altinn 3 Resource. 
+4. <!-- TODO: Set up of rights for the Altinn 3 Resource>
+<!-- TODO: Should we have links here to the different services used by service owners to create the different services? -->
+
```

Comment:
Consider adding step 5 and 6:
5. Go live with transition solution while encouraging consumers to migrate to Altinn Broker.
6. Once all consumers have migrated to Altinn Broker, decommision the Altinn 2 Broker Service.

Reply:
deone